internal/pkg/cli/flag.go

```diff
@@ -155,6 +155,7 @@ const (
 	pipelineFlagDescription = "Name of the pipeline."
 	profileFlagDescription  = "Name of the profile."
 	yesFlagDescription      = "Skips confirmation prompt."
+	execYesFlagDescription  = "Optional. Whether to install/update the Session Manager Plugin."
```

Comment:
meh no need to say the install, let's keep it concise:
```suggestion
	execYesFlagDescription  = "Optional. Whether to upgrade the Session Manager Plugin."
```